<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriverCreateTrip</class>
 <widget class="QWidget" name="DriverCreateTrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>298</width>
     <height>502</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="StartOfTrip">
      <property name="placeholderText">
       <string>Départ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="DestinationOfTrip">
      <property name="placeholderText">
       <string>Destination</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="NombrePlaces">
      <property name="placeholderText">
       <string>Places</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="carModelEdit">
      <property name="placeholderText">
       <string>Car Model</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="priceSpinBox">
      <property name="prefix">
       <string>$</string>
      </property>
      <property name="suffix">
       <string> USD</string>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CreateTrajectButton">
      <property name="text">
       <string>Create Traject</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backBtn">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Créer un trajet&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
